README.md

```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> This project is done as part of the 2nd day activities. It is made in order to help the student masters the tools and practice needed for all of followings projects in Microverse curriculum
+
+![screenshot](app_screenshot.png)
```

Comment:
- [ ] [Optional] Please replace the default README screenshot with a screenshot of your project